server/game/cards/characters/03/houseflorentknight.js

```diff
@@ -0,0 +1,43 @@
+const DrawCard = require('../../../drawcard.js');
+const _ = require('underscore');
+
+class HouseFlorentKnight extends DrawCard {
+    setupCardAbilities() {
+        this.forcedReaction({
+            when: {
+                onCardEntersPlay: (event, card) => card===this
```

Comment:
I need to have a stern word with eslint, it should be picking up things like this (the spacing around ===)